src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/CryptoConfigForwarder.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+
+namespace System.Security.Cryptography
+{
+    internal static class CryptoConfigForwarder
+    {
+        private static readonly Func<string, object> s_createFromName;
+
+        static CryptoConfigForwarder()
+        {
+            const string CryptoConfigTypeName =
+                "System.Security.Cryptography.CryptoConfig, System.Security.Cryptography.Algorithms";
+
+            const string CreateFromNameMethodName = "CreateFromName";
+
+            Type t = Type.GetType(CryptoConfigTypeName, true);
```

Comment:
Nit: could you name the bool argument so it's obvious what it is?  IIRC, it's throwOnError?

Reply:
I would if it'd be better to make it throwOnError:false, and then if you get back null, make s_createFromName be a delegate that throws a PlatformNotSupportedException.

Reply:
What's the motivation for the PNSE?  The type would only be missing if 1) we moved it (which we'd notice via the tests) or 2) a trimmer removed it.  If a trimmer removed it then the TypeLoadException seems more informative.

Of course, it could be a PNSE which carries the TypeLoadException or MissingMethodException as an innerException.

Reply:
I'm fine with either.